src/main/java/pl/orki/hackathon/webapp/band/control/BandService.java

```diff
@@ -37,11 +37,8 @@ public BandService(BandRepository bandRepository, UserRepository userRepository,
     }
 
     @Transactional
-    public Optional<Band> createBand(Band band, Long userId) {
-        return userRepository.findById(userId)
-                .map(user -> {
-                    band.setUser(user);
-                    return bandRepository.save(band);
-                });
+    public Optional<Band> createBand(Band band) {
+        userService.createUser(band.getUser());
+        return Optional.of(bandRepository.save(band));
```

Comment:
According to https://github.com/OrkiZMajorki/backend/pull/6/files?file-filters%5B%5D=.graphqls&file-filters%5B%5D=.java&hide-deleted-files=true#r500539084 maybe we should consider inverting approach. That means create a band, linked to user within the `userService`. It may be based on sended `role`.

Reply:
How then we will pass info about both venue and band?